config/prow/config.yaml

```diff
@@ -727,6 +727,32 @@ presets:
   - name: azure-ssh
     mountPath: /etc/azure-ssh
     readOnly: true
+- labels:
+    preset-azure-anonymous-pull: "true"
+  env:
+    - name: REGISTRY
+      value: capzci.azurecr.io
+- labels:
+    preset-azure-cred-only: "true"
```

Comment:
is the idea to reuse cred-only for the other jobs in the future? IMO it would make more sense to either 1) have one label preset `preset-azure-cred-anonymous-pull` for capz that has both creds and registry and not have any reuse between capz and other azure jobs like you had before or 2) have one `preset-azure-cred` label that is shared amongst all the Azure jobs (capz + others) that only contains creds and then `preset-azure-acr-anonymous-pull` + `preset-azure-acr` labels for registry/storage stuff

Reply:
I was shooting for 2. I thought this would be a good way to start the discussion about decomposing `preset-azure-cred`. I didn't want to do that in this PR. In this PR, I just wanted to setup the example for use in capz.

Reply:
great, that's my preference as well